integrations/influxdb_connector/influxdb-timestream-connector/src/lib.rs

```diff
@@ -87,33 +87,56 @@ async fn handle_multi_table_ingestion(
 
 pub async fn lambda_handler(
     client: &timestream_write::Client,
-    event: Request,
```

Comment:
`lambda_http::Request` structs can only come from an AWS service. The `lambda_http` crate is specifically for integrating with AWS services. However, integrations with AWS services (such as `AWS_PROXY`) only support synchronous invocation. Using `lambda_runtime::LambdaEvent<serde_json::Value>` structs instead mean we can support every kind of request we want to support, asynchronously or synchronously.